### PR TITLE
Finished translating :D

### PR DESCRIPTION
**Pull request by RyutoSetsujin**
I have translated the last lines in this document :D I feel so good

### CODE REVIEW

File: Story/Omnibus/st_010120_om.csv [hunk 1]
```diff
@@ -26,10 +26,10 @@ st_010130_0080#0,"""Please proceed to the planet Naberius."""
...
-al_010010_0060#0,"""そして、貴方を導くために<br>「マターボード」を渡したってわけだ。<br>ナベリウスに向かえってのもその一環。"""
-al_010010_0070#0,"""それは、「何か」を成し遂げるため。"""
-al_010010_0080#0,"""その「何か」が、きっとシオンの<br>「後悔」に繋がるものなんだろうけど<br>ここまでじゃ、まだ見えてこないかな。"""
+al_010010_0020#0,"""As usual, as I said, <br> something meaningless is still unclear. <br> But is it better than I am?"""
+al_010010_0030#0,"""I am also aware, through the creator <br> I was spoken to you a little. <br> <br> I was feeling regretted."""
+al_010010_0040#0,"""I think now, I thought that it might have apologized to me for me ... <br> Because the future and endings are visible."""
+al_010010_0050#0,"""...... I think that Xion, who spoke to you at this time saw me until the end of her own."""
+al_010010_0060#0,"""And I gave you the Matthew Board to guide you. <br> It's part of heading to Naverius."""
+al_010010_0070#0,"""That is something to achieve."""
+al_010010_0080#0,"""That "something" will surely lead to Xion 's repentance. <br> <br> It seems I can not see it yet."""
```
Comment (alama) [inline Story/Omnibus/st_010120_om.csv]: replace `"` with `\""`

Comment (RyutoSetsujin) [reply Story/Omnibus/st_010120_om.csv]: Ok.

**Review by RyutoSetsujin — COMMENTED** [review]
Sweet.

File: Story/Omnibus/st_010120_om.csv [hunk 2]
```diff
@@ -32,4 +32,4 @@ al_010010_0040#0,"""I think now, I thought that it might have apologized to me f
 al_010010_0050#0,"""...... I think that Xion, who spoke to you at this time saw me until the end of her own."""
 al_010010_0060#0,"""And I gave you the Matthew Board to guide you. <br> It's part of heading to Naverius."""
 al_010010_0070#0,"""That is something to achieve."""
-al_010010_0080#0,"""That "something" will surely lead to Xion 's repentance. <br> <br> It seems I can not see it yet."""
+al_010010_0080#0,"""That \"something\" will surely lead to Xion 's repentance. <br> <br> It seems I can not see it yet."""
```
Comment (alama) [inline Story/Omnibus/st_010120_om.csv]: double quotes, not single

**Comment by rebelclover** [comment]
Just some minor things regarding one of the lines.

./Story/Omnibus/st_010120_om.csv:al_010010_0060#0,"""And I gave you the Matthew Board to guide you. <br> It's part of heading to Naverius."""

- It's "Matter Board", not "Matthew Board"

- It's "Naberius", not "Naverius"

The last one is causing CI to fail.

**Pull request closed without merging**